crates/ruff_linter/src/rules/pylint/snapshots/ruff_linter__rules__pylint__tests__PLW1508_invalid_envvar_default.py.snap

```diff
@@ -1,6 +1,5 @@
 ---
 source: crates/ruff_linter/src/rules/pylint/mod.rs
-snapshot_kind: text
```

Comment:
Not sure why this was removed.

Reply:
Can you try updating your local `cargo insta` version running `cargo install cargo-insta --force`?